hal/lib/athena/handles/IndexedHandleResource.h

```diff
@@ -16,6 +16,9 @@
 #include "HAL/cpp/priority_mutex.h"
 #include "HandlesInternal.h"
 
+#define INDEXED_RESOURCE_INDEX_OUT_OF_RANGE -1
```

Comment:
Is there a reason this is a macro instead of const or constexpr? It's only used in C++ code from what I can tell.